src/nidigital/templates/session.py/fancy_get_site_pass_fail.py.mako

```diff
@@ -0,0 +1,18 @@
+<%page args="f, config, method_template"/>\
+<%
+    '''Forwards to _fetch()/_read() with a nicer interface'''
+    import build.helper as helper
+%>\
+    def ${f['python_name']}(${helper.get_params_snippet(f, helper.ParameterUsageOptions.SESSION_METHOD_DECLARATION)}):
+        '''${f['python_name']}
+
+        ${helper.get_function_docstring(f, False, config, indent=8)}
+        '''
+        result_list = self._get_site_pass_fail(site_list)
+
```

Comment:
I think you can get rid of this empty line since you didn't add one before the assert.